jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -288,6 +336,27 @@ object XGBoost extends Serializable {
       useExternalMemory, missing)
   }
 
+  /**
+   * This method trains an XGBoost model on a distributed collection of labeled training points.
+   * In each partition of the training RDD a booster is trained on the subset of data in that
+   * partition. The model is then updated globally using an all-reduce communication model across
+   * Spark workers.
+   *
+   * A [[RabitTracker]] is created to monitor the status of the all-reduce worker nodes and to
+   * verify that training completes successfully.
```

Comment:
RabitTracker is the scheduler for all workers not just for monitoring?

what do you mean by `Spark workers`?

The first four lines are talking about nearly the same thing with  the comments in `buildDistributedBoosters `

Reply:
Thanks for clarifying! I updated the wording.

I also updated the wording to indicate that all-reduce involves communication across all the Spark _tasks_, if that's clearer? 

I don't think they are talking about the same thing, `trainDistributed` is more general than `buildDistributedBoosters`, hence the method doc is more general.